packages/web-client/pages/login/index.tsx

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/button-has-type */
+import React, { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import { Button, Input } from '@skillfuze/ui-components';
+import LoginSVG from '../../assets/icons/login.svg';
+import { Header } from '../../components/header';
+import { Footer } from '../../components/footer';
+import AuthService from '../../services/auth.service';
+
+interface Props {
+  name: string;
+}
+
+const LoginPage: React.FC<Props> = () => {
+  const router = useRouter();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  useEffect(() => {
+    const token = localStorage.getItem('token');
+    if (token) {
+      router.push('/');
+    }
+  }, []);
+  const handleSubmit = async event => {
+    event.preventDefault();
+    const { token } = await AuthService.login({ email, password });
+    if (token) {
+      router.push('/');
+      localStorage.setItem('token', token);
+      const decodedUser = AuthService.decodeJWT(token);
+      AuthService.user = decodedUser;
+    }
+  };
+  return (
+    <div className="flex flex-col w-full h-screen">
+      <div className="flex w-full h-16">
```

Comment:
Why do you need to encapsulate the header in a div? if the header is not standalone we'll use the same exact div each time.